config/v201/component_schemas/standardized/InternalCtrlr.json

```diff
@@ -222,7 +222,7 @@
           "mutability": "ReadOnly"
         }
       ],
-      "default": "TLS_AES_256_GCM_SHA384,TLS_AES_128_GCM_SHA256",
+      "default": "TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256",
```

Comment:
We should also add the `TLS_CHACHA20_POLY1305_SHA256` here. It should be implemented, according to [TLS 1.3 RFC](https://datatracker.ietf.org/doc/html/rfc8446#section-9.1).

Reply:
cc: @hikinggrass (as you've added this file)

Reply:
I think we only added what the spec mentions what shall be supported here (A00.FR.318  says that a CSMS SHALL support at least):
```
TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
TLS_RSA_WITH_AES_128_GCM_SHA256
TLS_RSA_WITH_AES_256_GCM_SHA384
```

And A00.FR.319 mentions that a charging station shall support at least either
```
TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
and
TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
```

or 
```
TLS_RSA_WITH_AES_128_GCM_SHA256
and
TLS_RSA_WITH_AES_256_GCM_SHA384)
```

But we are free to add better ciphers here if available :smile:

Reply:
I would suggest keeping the default to the bare minimum required from OCPP perspective to provide the best interoperability on all charging stations. Maybe it is good to suggest in a adoption guide or something that users could add `TLS_CHACHA20_POLY1305_SHA256` if they want.